OpenOversight/app/templates/partials/incident_fields.html

```diff
@@ -1,15 +1,33 @@
-<tr>
-  <td>
-    <strong>Date</strong>
-  </td>
-  <td>{{ incident.date | display_date }}</td>
-</tr>
-{% if incident.time %}
+{% if not incident.occurred_at %}
+  {% if incident.date %}
+    <tr>
+      <td>
+        <strong>Date</strong>
+      </td>
+      <td>{{ incident.date | display_date }}</td>
+    </tr>
+  {% endif %}
+  {% if incident.time %}
+    <tr>
+      <td>
+        <strong>Time</strong>
+      </td>
+      <td>{{ incident.time | display_time }}</td>
+    </tr>
+  {% endif %}
+{% endif %}
+{% if incident.occurred_at %}
+  <tr>
+    <td>
+      <strong>Date</strong>
+    </td>
+    <td>{{ incident.occurred_at | local_date }}</td>
+  </tr>
   <tr>
     <td>
       <strong>Time</strong>
     </td>
-    <td>{{ incident.time | display_time }}</td>
+    <td>{{ incident.occurred_at | local_time }}</td>
```

Comment:
If I'm understanding this correctly, we are localizing the datetime to the user's own timezone here, right? I think it would make more sense to display the datetime in the department's timezone instead.

For example, if I were to look up an NYPD incident, I would want to know that the incident occurred around noon in NY, rather than 9am in Seattle.

Reply:
They translate accordingly and I think there's value in having them be discrete points in time, as opposed to them being relative points. Having them be discrete points in time also makes it so that its worth entering times for data collection things downstream from just this page.